src/java.desktop/share/classes/java/beans/Beans.java

```diff
@@ -64,6 +64,22 @@ public Beans() {}
      * <p>
      * Instantiate a JavaBean.
      * </p>
+     * The bean is created based on a name relative to a class-loader.
+     * This name should be a {@linkplain ClassLoader##binary-name binary name} of a class such as "a.b.C".
```

Comment:
```suggestion
     * This name should be a {@linkplain ClassLoader##binary-name binary name}
     * of a class such as "a.b.C".
```
Shall we wrap this line?

Should the names in the quotes be marked up with `{@code}` too?